samples/media/ndl/pkg_sample-ndl-webos4/appinfo.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What's this file for?

Reply:
Whoops, this is a custom run task in vscode. Sorry forgett to delete it.. 
It copied the compiled executeable from my build machine -> jumphost -> TV

Reply:
I dont really know where this comes from.. Deleted it. Maybe some autogenerated stuff, because at the first time I have build the completed project.